docs/docs/working-with-Etsy-in-Gatsby.md

```diff
@@ -0,0 +1,68 @@
+---
+Title: Working with Etsy in Gatsby
+---
+
+This guide explains how to begin working with the e-commerce site Etsy in Gatsby.
+
+## Etsy API
+
+Utilizing the Etsy API (Application Programming Interface), developers and non-developers can create apps for the web that pull in information such as BillCharge, CartListing, ShippingOption, and a list of other API references available to query. First, you must create a [developer account](<[https://www.etsy.com/developers/register](https://www.etsy.com/developers/register)>) with Etsy, confirm the account, and enable two-factor authentication. This will allow you to generate and receive your own API key. You will be asked for a short description of your app.
```

Comment:
```suggestion
Utilizing the Etsy API (Application Programming Interface), developers and non-developers can create apps for the web that pull in information such as BillCharge, CartListing, ShippingOption, and a list of other API references available to query. First, you must create a [developer account](https://www.etsy.com/developers/register) with Etsy, confirm the account, and enable two-factor authentication. This will allow you to generate and receive your own API key. You will be asked for a short description of your app.
```

They get interpreted as relative links, which leads to a 404.